src/peergos/server/Start.java

```diff
@@ -86,9 +90,87 @@
             Collections.emptyList()
     );
 
+    public static final Command BOOTSTRAP = new Command("bootstrap",
```

Comment:
maybe expand on documentation to highlight that the peergos.password should be specified

Reply:
+1

Reply:
done